app.js

```diff
@@ -101,7 +101,7 @@ app.get('/terms-and-conditions', function renderTerms(req, res) {
 app.use(require('./errors/'));
 
 
-/*eslint camelcase: 0*/
+/* eslint camelcase: 0*/
 app.listen(config.port, config.listen_host);
```

Comment:
You can also do `// eslint-disable-line camelcase` or `// eslint-disable-next-line camelcase